modules/sql-engine/src/test/java/org/apache/ignite/internal/sql/engine/planner/ImplicitCastsTest.java

```diff
@@ -346,18 +341,14 @@ public Stream<DynamicTest> testUpdate() {
 
                 checkStatement()
                         .table("t1", "c1", NativeTypes.INT32)
-                        .sql("UPDATE t1 SET c1 = 'abc'")
-                        .project("$t0", "CAST(_UTF-8'abc'):INTEGER NOT NULL"),
```

Comment:
I think this test case is either no longer necessary or should be updated to reflect that fact that numeric literals are checked during the validation phase.

Reply:
The same could be applied to the several test cases for `SELECT IN`

Reply:
I remove this test and prefer to stay with 'SELECT IN' tests ,they are quite small and seems useful